src/pages/Home.jsx

```diff
@@ -1,22 +1,107 @@
+              <h3 className='px-2'>Quality Guaranteed</h3>
+            </div>
+            <div className='flex items-center text-center justify-center bg-green-700 p-2.5 gap-2.5 rounded-lg w-3/5 lg:w-[463px] lg:h-[85px]'>
+              <Link
+                to='#services-section'
+                className='text-2xl lg:text-3xl text-white font-bold'
+              >
+                Our Services
+              </Link>
+            </div>
+          </div>
+        </div>
+      </div>
+    )
+  }
+
+  const renderMobile = () => {
+    return (
+      <>
+       <div className='flex bg-white py-2.5'>
+          <p className='font-normal font-header text-xl ml-auto mx-2'>
+            Tap to call!
+          </p>
+          <p className='font-normal font-header text-xl text-green-700 mx-2'>
+            123.456.7890
+          </p>
+        </div>
+      <div className="p-6 bg-white bg-[url('/public/images/hero.png')] bg-no-repeat bg-cover">
+        <div className='flex flex-col items-center justify-center p-6 bg-white bg-opacity-80 rounded-2xl'>
+          <div className='py-5 px-16 gap-2.5'>
+            <BaseHeading level={1} text='Yorkshire Woods Ground Crew' />
+            <hr className='w-[211px] h-px mx-auto mt-4 bg-black border-0 rounded'></hr>
+          </div>
+          <div>
+            <div className='w-[358px] h-[80px] px-8'>
+              <h2 className='font-normal italic text-center text-3xl text-zinc-500'>
+                Detroit&apos;s Finest Lawn Care
+              </h2>
+            </div>
+          </div>
+          <div className='gap-16'>
+            <div className='flex flex-col font-normal text-xl text-center mb-14 gap-2'>
+              <h3 className=''>Personalized Service</h3>
+              <h3 className=''>Jobs Of All Sizes</h3>
+              <h3 className=''>Quality Guaranteed</h3>
+            </div>
+            <div className='flex items-center text-center justify-center bg-green-700 py-3 px-10 rounded-lg'>
+              <Link
+                to='#services-section'
+                className='text-2xl text-white font-bold'
+              >
+                Our Services
+              </Link>
+            </div>
+          </div>
+        </div>
       </div>
-    </div>
-  )
+      </>
+    )
+  }
+
+  // The current width of the viewport
+  const [width, setWidth] = React.useState(window.innerWidth)
+  const breakpoint = 620
+  React.useEffect(() => {
+    /* Inside of a "useEffect" hook add an event listener that updates
+       the "width" state variable when the window size changes */
+    window.addEventListener('resize', () => setWidth(window.innerWidth))
+    /* passing an empty array as the dependencies of the effect will cause this
+      effect to only run when the component mounts, and not each time it updates.
+       We only want the listener to be added once */
+  }, [])
+  return width > breakpoint ? renderWeb() : renderMobile()
```

Comment:
observation(nb): This shouldn't be necessary, we should be able to use Tailwind's `sm`, `md`, etc prefixes for styling.